lib/metanorma/cli/commands/config.rb

```diff
@@ -0,0 +1,122 @@
+require "pathname"
+
+require "metanorma/cli/stringify_all_keys"
+
+module Metanorma
+  module Cli
+    module Commands
+      class Config < Thor
+        class_option :global, aliases: "-g", type: :boolean, default: false, desc: "Use global config"
+
+        desc "get NAME", "Get config value"
+        def get(name = nil)
+          config_path = Metanorma::Cli.config_path(options.global)
+          config = load_config(config_path)
+
+          if name.nil?
+            print File.read(config_path, encoding: "utf-8")
+          else
+            print(config.dig(*dig_path(name)) || "nil")
+          end
+        end
+
+        desc "set NAME VALUE", "Set config value"
+        def set(name, value = nil)
+          config_path = Metanorma::Cli.config_path(options.global)
+          config = load_config(config_path)
+
+          value = try_convert_to_bool(value)
+          ypath = dig_path(name)
+          deep_set(config, value, *ypath)
+
+          save_config(config, config_path)
+        end
+
+        desc "unset [name]", "Set config [value] for [name]"
+        def unset(name)
+          config_path = Metanorma::Cli.config_path(options.global)
+          config = load_config(config_path)
+
+          ypath = dig_path(name)
+          deep_unset(config, *ypath)
+
+          save_config(config, config_path)
+        end
+
+        def self.exit_on_failure?() true end
+
+        # priority:
+        # IDEAL: thor defaults -> global conf -> local conf -> env vars -> passed arguments
+        # ACTUAL: all arguments -> global conf -> local conf
+        # - thor doesn't provide to differentiate default values against passed args
+        # - thor doesn't allow to get all args available for current command
```

Comment:
Do we need to keep this comments here?

Reply:
@abunashir in my opinion it may be useful in the future, but I'm not strict about it